test/libsolidity/syntaxTests/inheritance/override/override_implemented_with_unimplemented_then_implemented.sol

```diff
@@ -0,0 +1,11 @@
+contract A {
+    function f() public virtual {}
+}
+abstract contract B is A {
+    function f() public virtual override;
+}
+contract C is B {
+    function f() public virtual override {}
+}
+// ----
+// TypeError 4593: (81-118): Overriding an implemented function with an unimplemented function is not allowed.
```

Comment:
This shows that the problem does not exist without multiple inheritance because we just do not allow unimplemented functions between implemented ones in the resolution chain.